spec/std/time/time_spec.cr

```diff
@@ -121,6 +121,34 @@ describe Time do
     end
   end
 
+  it "adds at time min" do
```

Comment:
The spec names are a bit vague. Maybe `it "checks for overflows"` or something more explicit to what's the expected outcome?

Reply:
Specs organization could probably be optimized (like group method specs in `describe` blocks) but that's for another PR...